[2025-08-06T15:59:07.701+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-06T15:59:07.709+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_transactions manual__2025-08-06T15:59:06.279140+00:00 [queued]>
[2025-08-06T15:59:07.713+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_s3_metadata_dag.extract_transactions manual__2025-08-06T15:59:06.279140+00:00 [queued]>
[2025-08-06T15:59:07.714+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-06T15:59:07.719+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_transactions> on 2025-08-06 15:59:06.279140+00:00
[2025-08-06T15:59:07.724+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=461) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-06T15:59:07.724+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_s3_metadata_dag', 'extract_transactions', 'manual__2025-08-06T15:59:06.279140+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/extract/mysql_to_s3_dag.py', '--cfg-path', '/tmp/tmpl2rmkuqi']
[2025-08-06T15:59:07.726+0000] {standard_task_runner.py:63} INFO - Started process 468 to run task
[2025-08-06T15:59:07.726+0000] {standard_task_runner.py:91} INFO - Job 45: Subtask extract_transactions
[2025-08-06T15:59:07.742+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-06T15:59:07.771+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_s3_metadata_dag.extract_transactions manual__2025-08-06T15:59:06.279140+00:00 [running]> on host 3bffe12d583b
[2025-08-06T15:59:07.840+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_s3_metadata_dag' AIRFLOW_CTX_TASK_ID='extract_transactions' AIRFLOW_CTX_EXECUTION_DATE='2025-08-06T15:59:06.279140+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-06T15:59:06.279140+00:00'
[2025-08-06T15:59:07.842+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-06T15:59:07.862+0000] {base.py:84} INFO - Using connection ID 'mysql_local' for task execution.
[2025-08-06T15:59:07.882+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:243 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-08-06T15:59:08.142+0000] {logging_mixin.py:188} INFO - Extracted transactions to /opt/***/temp/transactions.csv
[2025-08-06T15:59:08.142+0000] {logging_mixin.py:188} INFO - [LOG] Total row count extracted in the transactions is 83488
[2025-08-06T15:59:08.143+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/temp/transactions.csv
[2025-08-06T15:59:08.144+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-06T15:59:08.157+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_s3_metadata_dag, task_id=extract_transactions, run_id=manual__2025-08-06T15:59:06.279140+00:00, execution_date=20250806T155906, start_date=20250806T155907, end_date=20250806T155908
[2025-08-06T15:59:08.201+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-06T15:59:08.215+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-06T15:59:08.216+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
